Zend/zend.c

```diff
@@ -1275,6 +1279,10 @@ ZEND_API size_t zend_get_page_size(void)
 	SYSTEM_INFO system_info;
 	GetSystemInfo(&system_info);
 	return system_info.dwPageSize;
+#elif defined(__APPLE__)
+	/* Is in fact the global vm_page_size which is a kernel global
```

Comment:
If the purpose is to safe syscall, wouldn't it be better to cache the result like in https://github.com/apple-open-source-mirror/Libc/blob/5e566be7a7047360adfb35ffc44c6a019a854bea/gen/FreeBSD/getpagesize.c#L50-L64 for all platforms?